tardis/gui/widgets.py

```diff
@@ -256,9 +256,9 @@ def __init__(self, yamlconfigfile, parent=None):
                           'no_of_packets':[True, None],
                           'iterations':[True, None],
                           'black_body_sampling':{
-                                                    'start': '1 angstrom',
-                                                    'stop': '1000000 angstrom',
-                                                    'num': '1.e+6',
+                                                   'start': '1 angstrom',
+                                                   'stop': '1000000 angstrom',
+                                                   'num': '1.e+6',
```

Comment:
Don't make changes to files or code that are out of scope for your pull request.